diff --git a/wscript b/wscript
index 741fe1e5a4..ecf5dec972 100644
--- a/wscript
+++ b/wscript
@@ -769,11 +769,8 @@ video_output_features = [
                              "-fgnu89-inline"],
                      linkflags="-L/opt/vc/lib",
                      header_name="bcm_host.h",
-                     lib=['mmal_core', 'mmal_util', 'mmal_vc_client', 'bcm_host']),
-            # We still need all OpenGL symbols, because the vo_opengl code is
-            # generic and supports anything from GLES2/OpenGL 2.1 to OpenGL 4 core.
-            check_cc(lib="EGL", linkflags="-lGLESv2"),
-            check_cc(lib="GLESv2"),
+                     lib=['mmal_core', 'mmal_util', 'mmal_vc_client',
+                         'bcm_host', 'EGL', 'GLESv2', 'vchostif']),
         ),
     } , {
         'name': '--ios-gl',
