From 2ad49a821734cbe4e0578ef8eefbfaaffc00b75a Mon Sep 17 00:00:00 2001
From: Ryan Coe <bluemrp9@gmail.com>
Date: Sun, 8 Mar 2015 16:19:14 -0700
Subject: [PATCH 2/2] prevent adding local directories

Signed-off-by: Ryan Coe <bluemrp9@gmail.com>
---
 qb/config.libs.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qb/config.libs.sh b/qb/config.libs.sh
index efb88f5..4d7c6b2 100644
--- a/qb/config.libs.sh
+++ b/qb/config.libs.sh
@@ -6,9 +6,9 @@ check_switch_c NOUNUSED_VARIABLE -Wno-unused-variable
 add_define_make NOUNUSED_VARIABLE "$HAVE_NOUNUSED_VARIABLE"
 
 # There are still broken 64-bit Linux distros out there. :)
-[ -z "$CROSS_COMPILE" ] && [ -d /usr/lib64 ] && add_library_dirs /usr/lib64
+#[ -z "$CROSS_COMPILE" ] && [ -d /usr/lib64 ] && add_library_dirs /usr/lib64
 
-[ -z "$CROSS_COMPILE" ] && [ -d /opt/local/lib ] && add_library_dirs /opt/local/lib
+#[ -z "$CROSS_COMPILE" ] && [ -d /opt/local/lib ] && add_library_dirs /opt/local/lib
 
 DYLIB=-ldl;
 CLIB=-lc
-- 
2.3.2

